### PR TITLE
Island: Remove 32bit manual run options

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Remove 32bit manual run options. Related #1675 .

Add any further explanations here.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] ~Is the TravisCI build passing?~
* [x] ~Was the CHANGELOG.md updated to reflect the changes?~
* [x] ~Was the documentation framework updated to reflect the changes?~

## Testing Checklist

* [x] ~Added relevant unit tests?~
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [x] If applicable, add screenshots or log transcripts of the feature working

image

## Explain Changes

Are the commit messages enough? If not, elaborate.

### CODE REVIEW

**Review by VakarisZ — APPROVED** [review]

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOptions.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import AuthComponent from '../../AuthComponent';
 import {faLaptopCode} from '@fortawesome/free-solid-svg-icons/faLaptopCode';
 import InlineSelection from '../../ui-components/inline-selection/InlineSelection';
 import {cloneDeep} from 'lodash';
-import {faCloud, faExpandArrowsAlt} from '@fortawesome/free-solid-svg-icons';
+import {faExpandArrowsAlt} from '@fortawesome/free-solid-svg-icons';
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOptions.js:8]: Why is this change here? faCloud isn't used?

Comment (ilija-lazoroski) [reply monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOptions.js:8]: Yep. Eslint got it and I thought why not.
